fastcov.py

```diff
@@ -295,6 +303,10 @@ def exclMarkerWorker(fastcov_sources, chunk, exclude_branches_sw, include_branch
                     for key in ["lines", "branches"]:
                         if i in fastcov_data[key]:
                             del fastcov_data[key][i]
+                    if i in line_to_func:
+                        for key in line_to_func[i]:
+                            if fastcov_data["functions"][key]:
```

Comment:
I believe this will throw `KeyError` if key doesn't exist.
```suggestion
                            if key in fastcov_data["functions"]:
```